code/game/objects/items/devices/radio/radio.dm

```diff
@@ -439,8 +444,11 @@
 		return
 
 	var/list/spans = data["spans"]
-	SEND_SOUND(holder, 'sound/misc/radio_receive.ogg')
-	if(SPAN_COMMAND in spans)
+	if(COOLDOWN_FINISHED(src, audio_cooldown))
+		COOLDOWN_START(src, audio_cooldown, 0.5 SECONDS)
```

Comment:
That's a very short cooldown, do you think it'll suffice?

Reply:
It's a little longer than the regular audio stinger, and approximately as long as the audible part of the "important" stinger. The main goal was to prevent it from overlapping with itself which is accomplished by this solution.